rate_limiter.js

```diff
@@ -294,6 +329,41 @@ function createServiceCounter(serviceName) {
     return counter;
 };
 
+RateLimiter.prototype.killSwitchLimitForService =
+function killSwitchLimitForService(serviceName) {
+    var self = this;
+    // allow total RPS limit to kick in first
+    var rpsLimit = self.totalRpsLimit + 5;
+    if (self.serviceCounters[serviceName] &&
+        self.serviceCounters[serviceName].rpsLimit * 2 < rpsLimit) {
+        rpsLimit = self.serviceCounters[serviceName].rpsLimit * 2;
```

Comment:
Please lift the “rate limiter kill / drop factor” of 2 to a constant at the top of the file.